docs/acp/user/develop-an-app/develop-an-app.md

```diff
@@ -126,17 +149,24 @@ Every app requires default endpoints for the App Registry Service. This service
 To be able to use your Mini Framework as an app, add a new Spryker module:
 
 ```bash
-docker/sdk cli composer require spryker/app-kernel spryker/message-broker-aws spryker/propel-encryption-behavior 
+docker/sdk cli composer require 'spryker/app-kernel:dev-feature/without-encryption\ as\ 0.1.0  spryker/message-broker-aws'
 ```
 
 The `spryker/app-kernel` module transforms the Mini-Framework into an app. It provides SyncAPI schema and code for configuration and disconnection, as well as an AsyncAPI schema and code for the AppConfigure and AppDisconnect messages. 
 The `spryker/message-broker-aws` module installs the necessary plugins for sending and receiving messages. 
-After installing the modules, you need to configure them. See the [configuration example](https://github.com/spryker-projects/mini-framework/blob/examples/acp/hello-world/my-app-final/config/Shared/config_default.php#L28).
+
+### Step validation
+
+- Make sure that new packages are added in `composer.json` and composer command is executed without errors
 
 ## Set up the message broker
 
 Install and configure the message broker as described in [Set up the message broker](/docs/acp/user/develop-an-app/set-up-the-message-broker.html).
 
+### Step validation
+
+- Make sure that `spryker/message-broker` is installed and configured properly.
```

Comment:
How does one check it? The doc that is referenced does not have any validation steps either. We need to extend here or in the referenced doc.

Reply:
We gonna remove this section at all when prepare template for PBC skeleton